src/app.js

```diff
@@ -1 +1,58 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from "axios";
+
+// Op basis van de informatie uit de REST Countries API, haal je data op over _alle_ landen ter wereld
+
+// Pseudo code:
+// [x] schrijf asynchrone functie voor een axios get request voor end point "all countries"
+// - Schrijf de functie en roep deze in de global scope aan
+// - vang het resultaat van de get request in functie op in een variabel
+// - zet await in de variabel zodat code  pas inwerking treed als het resultaat van de get request ontvangen is
+// - vang de variabel op in een try and catch blok
+// - Log de variabel,check in de console wat nog meer gelogd moet worden voor het verkrijgen van de info die we nodig hebben, zet dit in de log
+// [x] Zet het resultaat van alle landen als lijst op de pagina
+// - We hebben van de endpoint een array met objecten ontvangen, map over de array en return de naam en vang op in een variabel.
+// - zet tag in index.html om naar toe te verwijzen en verwijs de variable daarnaar toe met HTML.
+
+
+async function fetchAllCountries() {
+    try {
+        const resultFetchAllcountries = await axios.get("https://restcountries.com/v2/all");
```

Comment:
Ik weet nooit helemaal wat ik moet zeggen op zich ziet het er allemaal wel gestructureerd uit de opdracht is nog niet af maar dat weet je zelf waarschijnlijk ook wel.

ten eerste probeer niet te veel lange namen te gebruiken en let op je naam conventie je gebruikt camelcase dus dan zou bij resultFetchAllcountries de C van Countries een hoofdletter moeten zijn.
Dit is belangrijk om in het vervolg geen spelfouten te maken en op die manier moeilijk vindbare problemen te krijgen.
result zou hier al volstaan omdat de variabele alleen bestaat binnen de scope van de functie.
Zou je buiten de functie ook gebruik maken van de deze array dan zou het weer nuttig zijn om aan te duiden wat voor result het is maar omdat je binnen de functie maar 1 result hebt is dat weer niet nodig en scheelt weer schrijfwerk.